cmd/svcat/instance/provision_cmd.go

```diff
@@ -73,9 +92,16 @@ func (c *provisonCmd) Validate(args []string) error {
 
 	var err error
 
-	c.params, err = parameters.ParseVariableAssignments(c.rawParams)
-	if err != nil {
-		return fmt.Errorf("invalid --param value (%s)", err)
+	if c.jsonParams != "" {
```

Comment:
Let's add a check that the user didn't specify both json and individual params at the same time.

You can add tests for your validation to https://github.com/kubernetes-incubator/service-catalog/blob/master/cmd/svcat/svcat_test.go#L42